marconi-chain-index/test/Spec/Golden/Cli/marconi-chain-index.help

```diff
@@ -0,0 +1,15 @@
+Missing: 
```

Comment:
We discussed this before and seemed to agree on not needing to test cli output. Which kind of bugs do these kind of tests help to prevent?

Reply:
Yes, we discuss this before, however, we had less CLI parameters at that time, and no plan to update marconi README. 
We plan on updating the Marconi README  with additional information, including usage and CLI parameters. That section of docs will be a link to the golden-file(s),  and will get updated as we introduce/update  parameters. 
Additionally,  the golden files, allow us to see what the user's see as they use the software, which I think is valuable.

Reply:
I don't know.. Why not paste one of these into the readme then with common flags? Right now there are ~10 new files which look like tests but actually are just documentation. Are we going to link to all of these? Also, we-the-developers can "see what the users see" by running the program instead of looking into any of the file (and first we'd need to find the correct file).

Reply:
agreed that 10 are excessive. Will you be OK with keeping one, which I think is what you're suggesting,  and we could link the out put of that test to the README?  I would much prefer avoid having to remember cut/paste when we update the CLIs

Reply:
@eyeinsky Golden files for the CLI are very useful for reviewing purposes. Whenever you make changes to the CLI, the reviewer can verify if anything has changed directly in Github, instead of running the software. User facing interface is very important and we need to make sure not to change anything unless absolutely required.

I'm fine with 10 IIF they each provide something valuable for verifying CLI changes.

Reply:
In this specific case, maybe just 2 are necessary? One with `--help` to show the interface CLI interface, and one which shows an error message whenever we provided an invalid CLI argument?